tests/qa/qatests

```diff
@@ -202,17 +207,25 @@ def main(token, test, modver, snapshot, jsonver, rlecver, osnick, light, quick,
     test_fname = os.path.join(HERE, f'{test}.json')
     if not os.path.exists(test_fname):
         raise click.ClickException(f"Invalid test name: {test}")
+    if light and vecsim:
+        raise click.ClickException("Invalid combination: light and vecsim")
 
     if modver == 'master':
         snapshot = True
     if rlecver is not None:
-        Test(token, test_fname, modver, snapshot, jsonver, rlecver, osnick, light).run()
+        if rlecver == 'master':
+            rs_versions = paella.flines(os.path.join(HERE, 'RS_VERSIONS'))
+            try:
+                rlecver = list(filter(lambda v: '100.0.0' in v, rs_versions))[0]
+            except:
+                raise click.ClickException("Cannot find master version (100.0.0) in RS_VERSIONS")
```

Comment:
what is the purpose of this fix?

Reply:
This is to allow you to ask for RLEC master version (i.e. `-r master`). Just a usability trick.
In the same manner, we can allow for `r 6.2.12` without the build number (not currently implemented but looks interesting).